test_scripts/test_setups.py

```diff
@@ -119,22 +122,19 @@ class TestPhase(Enum):
                                     'multi_400_8500_d560',
                                     'long_400_8500_longread'],  # Longslit read-out mode
                   'keck_lris_red_orig': ['long_300_5000'],
-                  'vlt_xshooter': ['VIS_1x1', 'VIS_2x1', 'VIS_2x2', 'VIS_manual', 'NIR'],
-                  'gemini_gnirs': ['32_SB_SXD', '10_LB_SXD'],
-                  'gemini_gmos': ['GS_HAM_R400_700', 'GS_HAM_R400_860', 'GN_HAM_R400_885'],
-                  'gemini_flamingos2': ['HK_HK', 'JH_JH'],
+                  'lbt_luci': ['LUCI-I', 'LUCI-II'],
+                  'lbt_mods': ['MODS1R_Longslit', 'MODS2R_Longslit'],
                   'magellan_mage': ['1x1'],
                   'magellan_fire': ['FIRE_Echelle', 'FIRE_Long'],
                   'mdm_osmos': ['MDM4K'],
+                  'mmt_binospec': ['Longslit_G600', 'Multislit_G270'],
+                  'mmt_mmirs': ['HK_zJ', 'J_zJ', 'K_K'],
                   'not_alfosc': ['grism4', 'grism19'],
                   'p200_dbsp_blue': ['600_4000_d55'],
                   'p200_dbsp_red': ['316_7500_d55'],
                   'p200_tspec': ['TSPEC'],
                   'vlt_fors2': ['300I'],
-                  'lbt_luci': ['LUCI-I', 'LUCI-II'],
```

Comment:
Similar comment here, why are these tests being removed from develop. I disagree with that.

Reply:
look again